mlflow/xgboost.py

```diff
@@ -130,7 +138,29 @@ def save_model(
                           model. The given example will be converted to a Pandas DataFrame and then
                           serialized to json using the Pandas split-oriented format. Bytes are
                           base64-encoded.
-
+    :param pip_requirements: Either an iterable of pip requirement strings
+        (e.g. ``["scikit-learn", "-r requirements.txt"]``) or the string path to a pip requirements
+        file on the local filesystem (e.g. ``"requirements.txt"``). If provided, this describes the
+        environment this model should be run in. If ``None``, a default list of requirements is
+        inferred from the current software environment. Requirements are automatically parsed and
+        written to a ``requirements.txt`` file that is stored as part of the model. These
+        requirements are also written to the ``pip`` section of the model's conda environment
+        (``conda.yaml``) file.
```

Comment:
This section is fantastic! Thank you! Can we include similar information for the `conda_env` parameter to indicate where dependency information is stored? I.e. indicate that pip requirements from `conda_env` are written to a pip `requirements.txt` file and that the full conda environment is written to `conda.yaml`. Can we do this for `log_model()` as well?

Reply:
Sure!